tests/admin/load-tests.php

```diff
@@ -9,25 +9,31 @@ class Admin_Load_Tests extends WP_UnitTestCase {
 
 	private static $demo_modules = array(
 		'js-and-css/demo-module-1' => array(
-			'name'         => 'Demo Module 1',
-			'description'  => 'This is the description for demo module 1.',
-			'experimental' => false,
-			'focus'        => 'js-and-css',
-			'slug'         => 'demo-module-1',
+			'name'            => 'Demo Module 1',
+			'description'     => 'This is the description for demo module 1.',
+			'experimental'    => false,
+			'enabled-notice'  => '',
+			'disabled-notice' => '',
+			'focus'           => 'js-and-css',
+			'slug'            => 'demo-module-1',
 		),
 		'something/demo-module-2'  => array(
-			'name'         => 'Demo Module 2',
-			'description'  => 'This is the description for demo module 2.',
-			'experimental' => true,
-			'focus'        => 'something',
-			'slug'         => 'demo-module-2',
+			'name'            => 'Demo Module 2',
+			'description'     => 'This is the description for demo module 2.',
+			'experimental'    => true,
+			'enabled-notice'  => '',
+			'disabled-notice' => '',
+			'focus'           => 'something',
+			'slug'            => 'demo-module-2',
 		),
 		'images/demo-module-3'     => array(
-			'name'         => 'Demo Module 3',
-			'description'  => 'This is the description for demo module 3.',
-			'experimental' => false,
-			'focus'        => 'images',
-			'slug'         => 'demo-module-3',
+			'name'            => 'Demo Module 3',
+			'description'     => 'This is the description for demo module 3.',
+			'experimental'    => false,
+			'enabled-notice'  => '',
+			'disabled-notice' => '',
+			'focus'           => 'images',
+			'slug'            => 'demo-module-3',
```

Comment:
See above, let's revert this in favor of exploring a proper API decoupled from this PR. Maybe this is the way we should go eventually, but I think it'll require more thinking and consideration first, so I'd rather not have this block the PR here.

Reply:
Done